xml/dm_registers.xml

```diff
@@ -65,19 +65,14 @@
 
             1: \Fcfgstraddrzero - \Fcfgstraddrthree registers hold the address of the
             configuration string.
-        <field name="versionhi" bits="3:2" reset="0" access="R">
-            Reserved for future use. Reads as 0.
-        </field>
-        <field name="versionlo" bits="1:0" reset="1" access="R">
-            00: There is no Debug Module present.
+        <field name="version" bits="3:0" reset="2" access="R">
+            0: There is no Debug Module present.
 
-            01: There is a Debug Module and it conforms to version 0.11 of this
+            1: There is a Debug Module and it conforms to version 0.11 of this
             specification.
 
-            10: There is a Debug Module and it conforms to version 0.13 of this
+            2: There is a Debug Module and it conforms to version 0.13 of this
             specification.
-
-            11: Reserved for future use.
```

Comment:
I'm tempted to put some general language somewhere that says any values not explicitly listed in the spec are reserved for future use.